packages/headless/src/features/facets/category-facet-set/category-facet-set-slice.ts

```diff
@@ -190,11 +193,10 @@ function buildCategoryFacetRequest(
   };
 }
 
-function convertCategoryFacetValueToRequest(
-  categoryFacetValue: CategoryFacetValue,
+function buildCategoryFacetValueRequest(
```

Comment:
I would prefer something more specific like `buildSelectedCategoryFacetValueRequest(...)`.

Reply:
I tried this, but I found the method name too long. I opted for the opposite approach, setting the state to idle. Good suggestion 👍